src/components/common/Forms/CreditCardForm/CreditCardForm.tsx

```diff
@@ -344,7 +344,7 @@ export default class CreditCardForm extends React.Component<Props, State> {
           onBlur={() => {
             this.validateField("creditCardNumber");
           }}
-          formatter={cardNumberInputFormatter}
+          // formatter={cardNumberInputFormatter}
```

Comment:
I think we should leave the formatter here. And also, we should still treat the edit / add forms as working with the full card number. Eventually we should have an endpoint for adding / editing a card, so having the formatter would still be helpful.
So for this, we should display the last 4 digits as a placeholder, like you've done here, but once the user types the number, it should behave like it used to, by keeping track of the entire card number and passing it to the relevant handlers for creating / editing cards.